crates/nu-protocol/src/shell_error.rs

```diff
@@ -115,6 +115,8 @@ pub enum ShellError {
         msg: String,
         #[label = "{msg}"]
         span: Span,
+        #[label = "encountered here"]
+        orig_span: Span,
```

Comment:
Mhh I think the span names are a bit confusing. To me `orig_span` sounds more like the origin of a Value.

Reply:
@sholderbach Better now?

Reply:
(done in https://github.com/nushell/nushell/pull/10036/commits/00b27604b38b16b35b0e97f9bdcf387b7a2be867)